### DISCUSSION TITLE
adding a column to dashboard example is not working

### DISCUSSION THREAD

**Discussion by hopezh** [quarto-dev/quarto-cli#8957] [Q&A] [answered]
### Description

For the [stock explorer example dashboard](https://github.com/jjallaire/stock-explorer-dashboard/blob/main/dashboard.qmd), I added one column with a simple printed message in between the stock chart and the table, and it works as shown below:

![2024-03-02 21 58 38](https://github.com/quarto-dev/quarto-cli/assets/23435810/72b1b03a-0356-4ac3-8338-5138fc24161f)

However, if I replace the code with an [example code from plotly](https://plotly.com/python/line-and-scatter/), nothing is rendered, and the new column is not shown at all: 

![2024-03-02 21 59 07](https://github.com/quarto-dev/quarto-cli/assets/23435810/4756bcc1-7557-4af8-983e-af81fe992f95)

May I ask if I missed anything important here that causing this issue? Thanks. 

The output of `quarto preview`:
```
Starting python3 kernel...Done

Executing 'dashboard.ipynb'
  Cell 1/9: ''...Done
  Cell 2/9: ''...Done
  Cell 3/9: ''...Done
  Cell 4/9: ''...Done
  Cell 5/9: ''...Done
  Cell 6/9: ''...Done
  Cell 7/9: ''...Done
  Cell 8/9: ''...Done
  Cell 9/9: ''...Done

pandoc 
  to: html
  output-file: dashboard.html
  template: >-
    C:\Users\USERNAME\AppData\Local\Programs\Quarto\share\formats\dashboard\template.html
  standalone: true
  section-divs: true
  html-math-method: mathjax
  wrap: none
  default-image-extension: png
  
metadata
  document-css: false
  link-citations: true
  date-format: long
  lang: en
  page-layout: custom
  title: Stock Explorer
  author: Ty Coon
  logo: logo.png
  theme:
    - yeti
    - custom.scss
  
Output created: dashboard.html

Watching files for changes
Browse at http://localhost:6406/
```

The output of my `quarto check`:
```
Quarto 1.4.550
[>] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.11: OK
      Dart Sass version 1.69.5: OK
      Deno version 1.37.2: OK
[>] Checking versions of quarto dependencies......OK
[>] Checking Quarto installation......OK
      Version: 1.4.550
      Path: C:\Users\USERNAME\AppData\Local\Programs\Quarto\bin
      CodePage: 936

[>] Checking tools....................OK
      TinyTeX: (not installed)
      Chromium: (not installed)

[>] Checking LaTeX....................OK
      Tex:  (not detected)

[>] Checking basic markdown render....OK

[>] Checking Python 3 installation....OK
      Version: 3.11.7 (Conda)
      Path: C:/Users/jizha/.conda/envs/ai/python.exe
      Jupyter: 5.7.1
      Kernels: python3

[>] Checking Jupyter engine render....OK

[>] Checking R installation...........(None)

      Unable to locate an installed version of R.
      Install R from https://cloud.r-project.org/
```

My system: 
- Windows 11
- Powershell 7.4.1

**Comment by mcanouil** [comment]
Do you solve whatever issue you were having that you closed the discussion?

You know that code from screenshots can't be used by anyone? (Also screen readers can't read it at all).

Please share actual code when asking for help regarding code.
Thank you for your cooperation.

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Reply by hopezh** [reply, marked as answer]
Thanks, @mcanouil. It was careless of me for posting the question without thorough checking. The issue I mentioned was caused by missing the `fig.show()` in the code... my bad :P Your advice on posting question is duly noted.

**Reply by mcanouil** [reply]
Glad you figured it out.
